modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -263,7 +263,10 @@ public enum IgniteFeatures {
     TASK_EVT_ATTRIBUTE_SUPPORT(69),
 
     /** Allows to node check connections with special heartbeat message to detect network issues rather quickly. */
-    TCP_COMMUNICATION_HEARTBEATS(70);
+    TCP_COMMUNICATION_HEARTBEATS(70),
+
+    /** Optimization of compression for encrypted snapshots. */
+    OPTIMIZED_COMPRESSED_ENCRYPTED_SNAPSHOTS(71);
```

Comment:
Forgot to add to 
https://ggsystems.atlassian.net/wiki/spaces/GG/pages/1192198276/Community+edition+features+list

Reply:
Added all missing features to the document